lib/src/facts/aix/networking_resolver.cc

```diff
@@ -12,6 +13,9 @@
 #include <netinet/in.h>
 #include <sys/kinfo.h>
 
+// Mark string for translation (a helper macro for simple cases)
+#define _(...) leatherman::locale::format(__VA_ARGS__)
```

Comment:
When Facter catches up to Leatherman 0.8.2, this unfortunate macro definition (here and in the other changed files) will be replaced with the sturdier: "using leatherman::locale::_"